src/components/PeopleTable.tsx

```diff
@@ -17,628 +27,95 @@
               </a>
             </span>
           </th>
-
           <th>
             <span className="is-flex is-flex-wrap-nowrap">
               Sex
-              <a href="#/people?sort=sex">
+              <a href="#/people?sort=name">
                 <span className="icon">
-                  <i className="fas fa-sort" />
+                  <i className="fas fa-sort-up" />
                 </span>
               </a>
             </span>
           </th>
-
           <th>
             <span className="is-flex is-flex-wrap-nowrap">
               Born
-              <a href="#/people?sort=born&amp;order=desc">
+              <a href="#/people?sort=name">
```

Comment:
The sort link for the 'Born' column incorrectly points to `sort=name`. It should be updated to reflect sorting by birth year, such as `sort=born`.